### PR TITLE
WIP: 239/docker changes: Fixed Docker Publish Failing

### PR DESCRIPTION
**Pull request by bhanutejags**
Fixed Docker Publish Failing.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -151,7 +151,8 @@ jobs:
       - run:
           name: Publish Docker Image to ECR
           command: |
-            IMAGE_TAG=${CIRCLE_TAG/v/''}
+            IMAGE_TAG=${CIRCLE_TAG}
+            IMAGE_TAG=$(echo $IMAGE_TAG | cut -c 2-)
```
Comment (hortongn) [inline .circleci/config.yml:155]: Can these 2 lines be combined to `IMAGE_TAG=$(echo $CIRCLE_TAG | cut -c 2-)` ?

Comment (bhanutejags) [reply .circleci/config.yml:155]: I think they can be, I'll make that change.

Comment (bhanutejags) [reply .circleci/config.yml]: Made the change.

**Review by hortongn — CHANGES_REQUESTED** [review]
@bhanutejags Please rebase and squash the commits

**Pull request closed without merging**